### PR TITLE
Module open api end points display 

### PR DESCRIPTION
**Pull request by lecunm**

### CODE REVIEW

File: frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<div id="{{id}}"></div>
+<div id="errorMessage" *ngIf="containError()">While loading these end points, we got an issue : {{openApiPathJson.errors}}</div>
```
Comment (loriepisicchio) [inline frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.html]:
```suggestion
<div id="errorMessage" *ngIf="containError()">Problem while loading end points : {{openApiPathJson.errors}}</div>
```

File: frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.spec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  beforeEach(() => {
+    fixture = TestBed.createComponent(OpenApiEndPointsComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
+});
```
Comment (loriepisicchio) [inline frontend/src/app/output/page-content/open-api-end-points/open-api-end-points.component.spec.ts]: Can you add some tests ? Not sure if it's possible to test Swagger display in nominal case, but you should at least be able to test error message display when it fails.

File: app/services/clients/OpenApiClient.scala [hunk 3]
```diff
@@ -97,4 +113,57 @@ object OpenApiClient {
       variable.find(_.name == "${openApi.json.url}").map(_.value)
     }
   }
+
+
+  def getSwaggerJsonInfos(swaggerJsonTree: JsObject): JsObject = {
+    val quotes = "\""
+    val swagger = Json.parse(s"{${quotes}swagger$quotes:${swaggerJsonTree.value("swagger")}}").as[JsObject]
```
Comment (notflorian) [inline app/services/clients/OpenApiClient.scala]:
```suggestion
    val swagger = Json.parse(s"""{"swagger":${swaggerJsonTree.value("swagger")}}""").as[JsObject]
```

File: app/services/clients/OpenApiClient.scala [hunk 4]
```diff
@@ -97,4 +113,57 @@ object OpenApiClient {
       variable.find(_.name == "${openApi.json.url}").map(_.value)
     }
   }
+
+
+  def getSwaggerJsonInfos(swaggerJsonTree: JsObject): JsObject = {
+    val quotes = "\""
```
Comment (notflorian) [inline app/services/clients/OpenApiClient.scala]: remove the variable `quotes` and use """ ... """ strings insteads

**Pull request merged**